README.md

```diff
@@ -32,7 +32,7 @@ And it runs on all platforms Rust supports, including Windows.
 * [Security](#security)
 * [FAQ](#faq)
 * [License](#license)
-* [Contributing](#contributing)
+* [Contributing](CONTRIBUTING.md).
```

Comment:
Why does this get a full-stop when the other bullets don't?

Reply:
Vim finger slip :)